trantor/utils/Funcs.h

```diff
@@ -30,4 +31,16 @@ inline uint64_t ntoh64(uint64_t n)
 {
     return hton64(n);
 }
+inline std::vector<std::string> strsplit(const std::string &s, const std::string &delimiter )
```

Comment:
this is implemented in drogon:
https://github.com/an-tao/drogon/blob/17b8c337f9fe6513e81fa3784bc7559cd7f8fcb7/lib/inc/drogon/utils/Utilities.h#L48-L51

Reply:
I see there is an implementation in drogon. However, trantor is not depended on drogon. Therefore, I reimplemented it and see drogon's splitString wrap on trantor implementation?
Or would it be good to move it to trantor?

Reply:
Oh, I see, this is ok, please use Hump nomenclature. (change strsplit to splitString)

Reply:
I will update that name and wrap the Drogon's functions if needed.

Reply:
Ok! thanks